Dockerfile-notebooks.dockerignore

```diff
@@ -3,7 +3,7 @@ services
 .env
 db/*.sqlite3
 logs
-tasks
+#tasks
```

Comment:
For the notebooks image we don't need the tasks - can we put this back in?

Reply:
Done